repro.in

```diff
@@ -118,29 +118,29 @@ function error() {
 }
 
 ##
-#  usage : lock( $fd, $file, $message, [ $message_arguments... ] )
+#  usage : elock( $fd, $file, $message, [ $message_arguments... ] )
```

Comment:
Nit: maybe add a comment/description - elock (write lock) and slock (read lock)?